src/components/cards/ClientTaskCard.tsx

```diff
@@ -35,89 +35,92 @@ export const ClientTaskCard = ({
-                  sx={{ width: '20px', height: '20px' }}
-                  variant={currentAssignee?.type === 'companies' ? 'rounded' : 'circular'}
-                />
-
-                <Typography
-                  variant="bodySm"
+            </Stack>
+            <Stack
+              direction="row"
+              alignItems="flex-start"
+              minWidth="fit-content"
+              columnGap={4}
+              justifyContent="space-between"
+            >
+              <Stack direction="row" alignItems="flex-end" minWidth="200px" columnGap={{ xs: 1, sm: 3 }}>
+                <Box
+                  minWidth={{ xs: '90px' }}
                   sx={{
-                    color: (theme) => theme.color.gray[600],
-                    textOverflow: 'ellipsis',
-                    whiteSpace: 'nowrap',
-                    overflow: 'hidden',
+                    flexDirection: 'row',
+                    alignItems: { sm: 'flex-end' },
+                    justifyContent: { sm: 'right' },
+                    display: 'flex',
                   }}
                 >
-                  {truncateText(
-                    (currentAssignee as IAssigneeCombined)?.name ||
-                      `${(currentAssignee as IAssigneeCombined)?.givenName ?? ''} ${(currentAssignee as IAssigneeCombined)?.familyName ?? ''}`.trim() ||
-                      'No Assignee',
-                    TruncateMaxNumber.SELECTOR,
+                  {task.dueDate && (
+                    <Typography variant="bodySm">
+                      <DueDateLayout date={task.dueDate} />
+                    </Typography>
                   )}
-                </Typography>
+                </Box>
+                <Stack direction="row" alignItems="flex-start" columnGap={1} sx={{ padding: '2px' }}>
+                  <Avatar
+                    src={currentAssignee?.iconImageUrl || currentAssignee?.avatarImageUrl || 'user'}
+                    alt={currentAssignee?.givenName || currentAssignee?.familyName || currentAssignee?.name}
+                    sx={{ width: '20px', height: '20px' }}
+                    variant={currentAssignee?.type === 'companies' ? 'rounded' : 'circular'}
+                  />
+
+                  <Typography
+                    variant="bodySm"
+                    sx={{
+                      color: (theme) => theme.color.gray[600],
+                      textOverflow: 'ellipsis',
+                      whiteSpace: 'nowrap',
+                      overflow: 'hidden',
+                    }}
+                  >
+                    {truncateText(
+                      (currentAssignee as IAssigneeCombined)?.name ||
+                        `${(currentAssignee as IAssigneeCombined)?.givenName ?? ''} ${(currentAssignee as IAssigneeCombined)?.familyName ?? ''}`.trim() ||
+                        'No Assignee',
+                      TruncateMaxNumber.SELECTOR,
+                    )}
```

Comment:
I think we are doing two things for the same purpose here. We are giving the above styling in `sx` prop which is exactly responsible for the truncate styling. We don't need to use `truncateText` function in this case.